tools/linter/adapters/ufmt_linter.py

```diff
@@ -2,20 +2,89 @@
 
 import argparse
 import concurrent.futures
+import fnmatch
 import json
 import logging
 import os
+import re
 import sys
 from enum import Enum
 from pathlib import Path
 from typing import Any, NamedTuple
 
+import isort
+from isort import Config as IsortConfig
 from ufmt.core import ufmt_string
 from ufmt.util import make_black_config
 from usort import Config as UsortConfig
 
 
 IS_WINDOWS: bool = os.name == "nt"
+REPO_ROOT = Path(__file__).absolute().parents[3]
+ISORT_WHITELIST = re.compile(
+    "|".join(
+        (
+            r"\A\Z",  # empty string
+            *map(
+                fnmatch.translate,
+                [
+                    # **
+                    "**",
+                    # .ci/**
+                    ".ci/**",
+                    # .github/**
+                    ".github/**",
+                    # benchmarks/**
+                    "benchmarks/**",
+                    # functorch/**
+                    "functorch/**",
+                    # tools/**
+                    "tools/**",
+                    # torchgen/**
+                    "torchgen/**",
+                    # test/**
+                    "test/**",
+                    # test/[a-c]*/**
+                    "test/[a-c]*/**",
+                    # test/d*/**
+                    "test/d*/**",
+                    # test/dy*/**
+                    "test/dy*/**",
+                    # test/[e-h]*/**
+                    "test/[e-h]*/**",
+                    # test/i*/**
+                    "test/i*/**",
+                    # test/j*/**
+                    "test/j*/**",
+                    # test/[k-p]*/**
+                    "test/[k-p]*/**",
+                    # test/[q-z]*/**
+                    "test/[q-z]*/**",
+                    # torch/**
+                    "torch/**",
+                    # torch/_[a-c]*/**
+                    "torch/_[a-c]*/**",
+                    # torch/_d*/**
+                    "torch/_d*/**",
+                    # torch/_[e-h]*/**
+                    "torch/_[e-h]*/**",
+                    # torch/_i*/**
+                    "torch/_i*/**",
+                    # torch/_[j-z]*/**
+                    "torch/_[j-z]*/**",
+                    # torch/[a-c]*/**
+                    "torch/[a-c]*/**",
+                    # torch/d*/**
+                    "torch/d*/**",
+                    # torch/[e-n]*/**
+                    "torch/[e-n]*/**",
+                    # torch/[o-z]*/**
+                    "torch/[o-z]*/**",
+                ],
+            ),
+        )
+    )
+)
```

Comment:
I found a way to split the one stack into multiple ones after the first PR is merged.